ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml

```diff
@@ -1172,7 +1169,7 @@ tests:
     dependency_overrides:
       OO_INDEX: quay.io/openshift-qe-optional-operators/aosqe-index:v4.13
     env:
-      E2E_RUN_TAGS: '@gcp-ipi and not @storage'
+      E2E_RUN_TAGS: not @storage
```

Comment:
missing ''?

Reply:
Nope.   `make update` does not allow `'xxx'` when the first char is not @

Reply:
ok